### PR TITLE
Return empty recommendations for unknown client

### PR DESCRIPTION
**Pull request by maurodoglio**

### CODE REVIEW

File: taar/recommenders/recommendation_manager.py [hunk 1]
```diff
@@ -34,6 +34,8 @@ def __init__(self, profile_fetcher=None, recommenders=None):
     def recommend(self, client_id, limit):
         # Get the info for the requested client id.
         client_info = self.profile_fetcher.get(client_id)
+        if client_info is None:
+            return ()
```
Comment (Dexterp37) [inline taar/recommenders/recommendation_manager.py]: Shouldn't this be an empty list `[]`?

File: tests/test_recommendation_manager.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+from taar.profile_fetcher import ProfileFetcher
+from taar.recommenders import RecommendationManager
+from taar import hbase_client
+
+
+def test_none_profile_returns_empty_list(monkeypatch):
```
Comment (Dexterp37) [inline tests/test_recommendation_manager.py]: Thanks for doing this! 🎉🎉

File: taar/recommenders/recommendation_manager.py (hunk 1 above)
Comment (maurodoglio) [reply taar/recommenders/recommendation_manager.py]: Yea let's keep it consistent

**Comment by maurodoglio** [comment]
@Dexterp37 I added a few commits to fix some style issues and configure automatic releases to pypi. Can you please have a quick look?

**Pull request merged**